src/assets/translations/sw-SW.json

```diff
@@ -2413,7 +2413,8 @@
       "Navigation": "Urambazaji",
       "Notifications": "Arifa",
       "Organization": "Shirika",
-      "Payments": "Malipo",
```

Comment:
I reckon here you have modified the wrong translation key!

Reply:
fixed